helm-chart/binderhub/templates/configmap.yaml

```diff
@@ -19,6 +19,19 @@ data:
   binder.repo2docker-image: {{ .Values.repo2dockerImage | quote }}
   binder.hub-url: {{ .Values.hub.url | quote }}
   binder.base_url: {{ .Values.baseUrl | quote }}
+  {{ if .Value.use_as_federation_portal -}}
+  binder.use_as_federation_portal: {{ .Value.use_as_federation_portal }}
+  {{- end }}
+  binder.cannonical_address: {{.Values.cannonical_address | quote }}
```

Comment:
Is there a default specified for this in values.yaml? If not this should also have a guard or have a default specified.